.pipelines/templates/.builder-release-template-windows.yaml

```diff
@@ -22,8 +22,8 @@ steps:
           sigImageName="${SIG_IMAGE_NAME_PREFIX}-${WINDOWS_SKU}-gen2"; \
           echo "##vso[task.setvariable variable=GEN2_SIG_FOR_PRODUCTION]False"; \
         else
-          sigImageName="windows-${WINDOWS_SKU}"; \
-          sigGalleryName="WS2022Gen2Gallery$(date +"%y%m%d")"; \          
+          sigImageName="windows-${WINDOWS_SKU}-gen2"; \
+          sigGalleryName="WS2022Gen2Gallery$(date +"%y%m%d%m%H%M")"; \          
```

Comment:
Please add %s to avoid the confliction

Reply:
And please publish the pr for code review

Reply:
Adding %S so that gallery name looks like "WS2022Gen2Gallery220909022841" instead of "WS2022Gen2Gallery22090902291662690573".

Reply:
Will it not fail when two PRs are submitted at the same time in seconds?

Reply:
Sorry are you suggesting adding more % formatting suffix to the gallery name? I think it's highly unlikely that two PRs are simultaneously submitted either manually or automatically. Besides, there's some time from the time of submission untill the code `sigGalleryName="WS2022Gen2Gallery$(date +"%y%m%d%H%M%S")"; ` gets executed?

Reply:
%S may be enough, but I really do not want to waste time to investigate the failures for the same sigGalleryName

Reply:
> %S may be enough, but I really do not want to waste time to investigate the failures for the same sigGalleryName

I prefere to use %s but not %S for this reason.

Reply:
True. It seems like the packer error message was not concrete or sensible enough. "Do not rat hole a problem. If something fundamental issues found in the"

According to [this bug](https://github.com/hashicorp/packer/issues/8653)
"Our issue ended up being that our concurrent builds were sharing a managed_image_name and shared_image_gallery_destination.image_version".

So changing the sig gallery name and sig image name to unique values in [this](https://github.com/Azure/AgentBaker/pull/2153/commits/708693f780a58e1337aed3f82ed27fdcb913dacc).

https://github.com/hashicorp/packer-plugin-azure/issues/41

Reply:
I think that https://github.com/hashicorp/packer/issues/8653 may not be the root cause since you have use $random to generate different galleryname in your failed pipelines.

Reply:
Tested three times in the PR check-in pipeline. I think that post is trying to say it might be caused by the same managed_image_name and image version. Managed image name, in our case, is `sigImageName="windows-${WINDOWS_SKU}-gen2"`. So to avoid confusion, $RANDOM is added at the back of both image name and gallery name.